WowPacketParser/Misc/ClientVersion.cs

```diff
@@ -1084,6 +1084,8 @@ public static ClientVersionBuild GetVersionDefiningBuild(ClientVersionBuild buil
                 case ClientVersionBuild.V1_14_3_48611:
                 case ClientVersionBuild.V1_14_3_49229:
                 case ClientVersionBuild.V1_14_3_49821:
+                case ClientVersionBuild.V1_14_4_51146:
+                case ClientVersionBuild.V1_14_4_51535:
```

Comment:
Isn't 1.14.4 rebased upon Dragonflight? It ought to be grouped with 3.4 not with 2.5, or it won't inherit packet handlers from Dragonflight.

Reply:
well 3.4.0 was SL
3.4.3 is DF.

It's a mess tbh dont mess with all the inherit packet handlers :S

Reply:
I am doing it properly in last commit